server/BagAPI/Controllers/SearchController.cs

```diff
@@ -0,0 +1,103 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using BagMyFoodAPI.DTO;
+using Newtonsoft.Json;
+using System.Net;
+using BagMyFoodAPI.Models;
+
+namespace BagMyFoodAPI.Controllers
+{
+    [Route("api/[controller]")]
+    public class SearchController : Controller{
```

Comment:
instead of a search controller it should be the thing we're searching for. A product perhaps like this data model
http://www.databaseanswers.org/data_models/grocery_store/index.htm